freshmaker/errata.py

```diff
@@ -495,6 +495,27 @@ def get_cve_affected_rpm_nvrs(self, errata_id):
 
         return list(nvrs)
 
+    def get_cve_affected_build_nvrs(self, errata_id):
+        """ Get module build nvrs which are affected by the CVEs in errata
+
+        :param errata_id: Errata advisory ID, e.g. 25713.
+        :type errata_id: str or int
+        :return: a list of strings each of them is a module build nvr
+        :rtype: list
+        """
+        data = self._errata_rest_get(f"/erratum/{errata_id}/builds_by_cve")
+        nvrs = set()
+        for data_by_product in data.values():
+            for product_data in data_by_product.values():
+                for build in product_data.get("builds", []):
+                    for build_info in build.values():
+                        if not build_info.get("is_module"):
+                            continue
+                        nvr = build_info.get("nvr")
+                        nvrs.update([nvr])
```

Comment:
```suggestion
                        nvrs.add(nvr)
```

Reply:
Sure. I was imitating an existing method. add() is good enough.